### PR TITLE
X509 OID for attestation transparency for sgx_pkix. HRST-33.

### PR DESCRIPTION
**Pull request by arai-fortanix**
This adds an X509 OID for attestation transparency. This change was previously made ont he sgx_pkix 0.1.x branch (for sgx_pkix version 0.1.4). This change adds the new OID on the master branch for sgx_pkix version 0.2.2, for when we migrate roche to the latest releases of the crates that belong to rust-sgx.

### CODE REVIEW

**Review by Taowyoo — APPROVED** [review]
LGTM

**Comment by arai-fortanix** [comment]
This is failing due to https://github.com/fortanix/rust-sgx/issues/433, so I will wait until that is merged and I'll rebase.

**Comment by arai-fortanix** [comment]
I rebased with Yuxiang's fix to disable building isgx-pe2sgx, so hopefully CI will pass.

**Pull request merged**
